ci/dart.yml

```diff
@@ -8,15 +8,30 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
+    # Note that this workflow uses the latest stable version of the Dart SDK.
+    # Docker images for other release channels - like dev and beta - are also
+    # available. See https://hub.docker.com/r/google/dart/ for the available
+    # images.
     container:
       image:  google/dart:latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Install dependencies
-      run: pub get
-    - name: Run tests
-      run: pub run test
+      - uses: actions/checkout@v2
+
+      - name: Install package dependencies
```

Comment:
I've found it useful to add this step at the first step:

```
      - name: Report version
        run: dart --version
```

Reply:
added